cfg_mgmt/util.py

```diff
@@ -496,23 +495,23 @@ def process_cfg_queue_and_persist_in_repo(
 
 def scan_result_group_collection(
     results: tuple[gcm.CfgScanResult],
+    severity: gcm.Severity,
+    policy_violation: cmm.CfgStatusEvaluationAspects,
+    issue_type: str,
 ) -> gcm.ScanResultGroupCollection:
     def classification_callback(result: gcm.CfgScanResult) -> gcm.Severity:
-        return gcm.Severity.HIGH
+        return severity
 
     def findings_callback(result: gcm.CfgScanResult) -> bool:
-        if not result.evaluation_result.nonCompliantReasons:
-            return False
+        policy_violations = result.evaluation_result.nonCompliantReasons
+        if policy_violation in policy_violations:
```

Comment:
see comment w.r.t. issue-type - merge enum w/ labels, please.

Reply:
As discussed, keep labels and compliance violations duplicated (due to circular dependency) and have a consistent mapping (rather than passing around).